code/modules/mob/living/carbon/human/death.dm

```diff
@@ -79,6 +79,18 @@
 	if(is_hivehost(src))
 		var/datum/antagonist/hivemind/hive = mind.has_antag_datum(/datum/antagonist/hivemind)
 		hive.destroy_hive()
+	if(HAS_TRAIT(src, TRAIT_DROPS_ITEMS_ON_DEATH))
+		var/list/contents = get_contents()
```

Comment:
This should only throw top level contents (src.contents) rather than all the recursive contents of a mob. Much better for a backpack to be dropped than have all the bullets in the box inside the backpack be thrown everywhere